src/main/java/org/broadinstitute/consent/http/db/ElectionDAO.java

```diff
@@ -148,9 +148,9 @@ void bulkCancelOpenElectionByReferenceIdAndType(@Bind("electionType") String ele
             "     AND electionView.referenceId = e.referenceId " +
             "     AND LOWER(e.electionType) = LOWER(:type) " +
             "     AND e.finalAccessVote = :vote " +
-            "     AND LOWER(e.status) != 'canceled' " +
+            "     AND LOWER(e.status) not in ('canceled', 'closed') " +
```

Comment:
Can elections have a status = "Final"?

Reply:
Yes, and final elections are still vote-able. It is not intuitive, and I don't think I understood that until recently. Closed and canceled are the only election types that you cannot vote on. Final represents a legacy middle state where there were several votes on an election, including member and chair votes, but before it was really, really, really finally voted on. We don't use the final election status in the V2 voting process

### Legacy election status paths:
* Open -> Canceled
* Open -> Closed
* Open -> Final
* Open -> Final -> Canceled
* Open -> Final -> Closed
* Closed -> Open
* Canceled -> Open

### Legacy election status paths:
* Open -> Canceled
* Open -> Closed
* Closed -> Open
* Canceled -> Open